doc/sphinx/addendum/type-classes.rst

```diff
@@ -323,7 +323,7 @@ Summary of the commands
 
    .. cmdv:: Existing Class @ident
 
-      This variant declares a class a posteriori from a constant or
+      This variant declares a class from a constant or
```

Comment:
```suggestion
      This variant declares a class from a previously declared constant or
```

This reintroduces the distinction that was previously marked with "a posteriori".